repository/agenda-general.js

```diff
@@ -150,6 +155,10 @@ const setAgendaStatus = async (agendaURI, statusURI) => {
   INSERT {
     ${sparqlEscapeUri(agendaURI)} besluitvorming:agendaStatus ${sparqlEscapeUri(statusURI)} ;
       dct:modified ${sparqlEscapeDateTime(modifiedDate)} .
+    ${sparqlEscapeUri(agendaStatusActivity)} a ext:AgendaStatusActivity ;
```

Comment:
Dit heeft ook een `mu:uuid` nodig, anders kan mu-cl-resources het niet oppikken. Zelfde uuid als degene die je in de URI zet.